build-content-management-system.mdx

```diff
@@ -0,0 +1,451 @@
+
+![View uploads and content schema](/images/cms-remix-xata-04.png)
+
+Let’s move on to adding relevant columns in the tables you've just created.
+
+## Creating the Schema
+
+In the `uploads` table, you want to store all the images only (and no other attributes) so that you can create references to the same image object again, if needed.
+
+Proceed with adding the column named `image`. This column is responsible for storing the `file` type objects. In our case, the `file` type object is for images, but you can use this for storing any kind of blob (e.g. PDF, fonts, etc.) that’s sized up to 1 GB.
+
+First, click **+ Add column** and select **File**.
+
+![Add a column](/images/cms-remix-xata-05.png)
+
+Set the column name to `photo` and to make files public (so that they can be shown to users when they visit the image gallery), check the **Make files public by default** option.
+
+![Make files public by default](/images/cms-remix-xata-06.png)
+
+In the `content` table, we want to store the attributes such as content’s unique slug (the path of the url where content will be displayed), title, author name, author’s image with it’s dimensions, and content’s og image with it’s dimensions.
+
+Proceed with adding the column named `slug`. It is responsible for maintaining the uniqueness of each content that gets created. Click **+ Add a column**, select `String` type and enter the column name as `slug`. To associate a slug with only one content, check the `Unique` attribute to make sure that duplicate entries do not get inserted.
+
+![Add slug column](/images/cms-remix-xata-07.png)
+
+In similar fashion, create `title`, `author_name`, `author_image_url`, `og_image_url`, `author_image_w`, `author_image_h`, `og_image_w`, `og_image_h` as `String` type (but not `Unique`).
+
+Great, you can also store the user `content` as `Text` type. While `String` is a great default type, storing more than 2048 characters would require you to switch to the `Text` type. Read more about the limits in [Xata Column limits](https://xata.io/docs/rest-api/limits#column-limits).
+
+Lovely! With all that done, the final schema shall be as below 👇🏻
+
+![Add more CMS columns](/images/cms-remix-xata-08.png)
+
+## Setting up the project
+
+Clone the app repository and follow this tutorial; you can fork the project by running the following command:
+
+```bash
+git clone https://github.com/rishi-raj-jain/remix-wysiwyg-litellm-xata-vercel
+cd remix-wysiwyg-litellm-xata-vercel
+npm install
+```
+
+## Configure Xata with Remix
+
+To seamlessly use Xata with Remix, install the Xata CLI globally:
+
+```bash
+npm install @xata.io/cli -g
+```
+
+Then, authorize the Xata CLI so it is associated with the logged in account:
+
+```bash
+xata auth login
+```
+
+![Create new API key](/images/cms-remix-xata-09.png)
+
+Great! Now, initialize your project locally with the Xata CLI command:
```

Comment:
This needs a bit of clarification because his DB URL is hardcoded.

```suggestion
Great! Now, initialize your project locally with the Xata CLI command. In this command, you will need to use the database URL for the database that you just created. You can copy the URL from the Settings page of the database.
```